test/category.spec.js

```diff
@@ -1,152 +1,275 @@
 import chai from 'chai';
 import chaiHttp from 'chai-http';
+import sinonChai from 'sinon-chai';
 import updateVerifiedStatus from './helpers/updateVerifiedStatus';
+import assignRole from './helpers/assignRole';
 import app from '../index';
 import {
+  user,
   validCategory,
   invalidCategoryTooShort,
   invalidCategoryDuplicate,
+  validCategoryEdit
 } from './helpers/categoryDummyData';
 
-import { newUser2 } from './helpers/userDummyData';
+import { User, Category } from '../models';
 
 chai.use(chaiHttp);
+chai.use(sinonChai);
 const { expect } = chai;
+let userToken;
+let categoryId;
 
-
-describe('CREATE CATEGORY', () => {
-  let userToken;
-
-  before((done) => {
-    chai
+describe('TESTIN THE CATEGORY FEATURE', () => {
```

Comment:
@chingsley why's this and a few others in all caps?

Reply:
Also, I can't see a test for the super admin user

Reply:
I have included a test for creating a category by super admin. I implemented this by changing the role of a regular user to super admin. I didn't want to use the actual super admin's credentials (the team's email and password) in the test, as it might constitute a security risk.

Reply:
I have changed the caps to lower case, except the first letters